### ISSUE TITLE
setup.cfg update

### ISSUE THREAD

**Issue by subbyte** [IBM/firepit#4]
When running `pytest` with Python 3.9.5, get the following warning:
```
========================================================================================================== warnings summary ===========================================================================================================
../../venv/kestrel39/lib/python3.9/site-packages/_pytest/config/__init__.py:1233
  /workspace/venv/kestrel39/lib/python3.9/site-packages/_pytest/config/__init__.py:1233: PytestConfigWarning: Unknown config option: collect_ignore
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=================================================================================================== 106 passed, 1 warning in 1.88s ====================================================================================================
```

This seems to be a cookiecutter template issue:
https://github.com/audreyfeldroy/cookiecutter-pypackage/issues/608

Consider removing the following lines in `setup.cfg`:
```
[tool:pytest]
collect_ignore = ['setup.py']
```

**Comment by pcoccoli** [comment]
Interesting...I don't see this with python 3.9.2.

**Comment by pcoccoli** [comment]
Should be fixed by bec062b
